python/ray/experimental/async_plasma.py

```diff
@@ -10,6 +10,28 @@ class PlasmaObjectFuture(asyncio.Future):
     pass
 
 
+def _complete_future(eventHandler, ray_object_id):
+    # TODO(ilr): Consider race condition between popping from the
+    # waiting_dict and as_future appending to the waiting_dict's list.
+    logger.debug(
+        "Completing plasma futures for object id {}".format(ray_object_id))
+    obj = ray.get(ray_object_id, timeout=0.1)
```

Comment:
Should this be a flag?

Reply:
I think timeout is 0 is appropriate because it will ensure we check the object is there only once. Since our precondition is that the object already exists, we the timeout error should happen when that precondition breaks

Reply:
Okay, I'l switch this to 0.